software/pretix.yml

```
@@ -5,6 +5,9 @@ licenses:
   - Apache-2.0
 platforms:
   - Python
+  - Docker
+  - K8S
```

```suggestion
  - K8S
```

I see no mention of K8S in the documentation

---

Still pending a merge: https://github.com/pretix/pretix/pull/4011
But here is the helm chart artifacthub: https://artifacthub.io/packages/helm/techwolf12/pretix

---

Sure, but it can only be added when `doc/admin/installation/kubernetes.rst` is merged into the upstream documentation, please remove this platform for the time being. Thanks